src/components/App/App.tsx

```diff
@@ -1,21 +1,137 @@
-import React from 'react';
-import { withTranslation, WithTranslation } from 'react-i18next';
-import s from './App.scss';
+import React, { Fragment } from 'react';
+import {
+  Layout,
+  Cell,
+  Page,
+  Breadcrumbs,
+  Button,
+  Card,
+  FormField,
+  Input,
+  Heading,
+  Dropdown,
+  AddItem,
+  IconButton,
+  Text,
+  Box,
+} from 'wix-style-react';
+import DeleteSmall from 'wix-ui-icons-common/DeleteSmall';
 
-interface AppProps extends WithTranslation {}
-
-class App extends React.Component<AppProps> {
-  render() {
-    const { t } = this.props;
-
-    return (
-      <div className={s.root}>
-        <h2 className={s.title} data-hook="app-title">
-          {t('app.title', { who: 'Yoshi' })}
-        </h2>
-      </div>
-    );
-  }
-}
-
-export default withTranslation()(App);
+export default () => (
+  <Fragment>
+    <Box padding={8} backgroundColor="#f0f4f7">
+      <Layout>
+        <Cell>
+          <Layout>
+            <Cell>
+              <Page.Header
+                title={<Heading appearance="H1">WSR Form</Heading>}
```

Comment:
```suggestion
                title='WSR Form'
```

It is implemented already as H1 when providing a string. no need to wrap it  with `<Heading/>`